src/test-helpers/queries/findThoughtByText.ts

```diff
@@ -0,0 +1,9 @@
+import { findAllByText, screen } from '@testing-library/dom'
+
+/**
+ * Gets a thought that matches the given value.
+ */
+export const findThoughtByText = async (value: string, container?: HTMLElement) => {
+  const thoughtNodes = await (container ? findAllByText(container!, value, { exact: true }) : screen.findAllByText(value, { exact: true }))
```

Comment:
I recommend splitting long ternaries into multiple lines for better readability.